saturn-datepicker/src/datepicker/tsconfig-build.json

```diff
@@ -1,15 +0,0 @@
-{
-  "extends": "../tsconfig-build",
-  "files": [
-    "public-api.ts",
-    "../typings.d.ts"
-  ],
-  "angularCompilerOptions": {
-    "annotateForClosureCompiler": true,
-    "strictMetadataEmit": true,
-    "flatModuleOutFile": "index.js",
-    "flatModuleId": "@angular/material/datepicker",
```

Comment:
NOTE: The library had a invalid flatModuleId, so I cleanned up all related code and also removed the invalid bazel build configuration.